Project-Sloth/ps-realtor
# hotfix: trim-error

# Overview
Fix trim issue when having 0 properties and pressing on the add property button

# Details
Fixing trim issue on ps realtor, sometimes the string is undefined, but if you force to call trim function it will throw error and messed up the UI, so adding a optional chaining operator (`?`) prevent it to throw error 

# UI Changes / Functionality
Before fix:
https://streamable.com/30jzx3

After fix:
https://streamable.com/1q55yv

# Testing Steps
*Provide a list of repro steps on how to test that your changes are valid.*

- [✅] Did you test the changes you made?
- [✅ ] Did you test core functionality of the script to ensure your changes do not regress other areas?
- [❌] Did you test your changes in multiplayer to ensure it works correctly on all clients? (Don't have friend and my pc too potato😔)
